pkg/controller/codestatus/expire.go

```diff
@@ -51,3 +52,46 @@ func (c *Controller) HandleExpire() http.Handler {
 			})
 	})
 }
+
+func (c *Controller) HandleExpirePage() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		var code *database.VerificationCode
+
+		session := controller.SessionFromContext(ctx)
+		if session == nil {
+			controller.MissingSession(w, r, c.h)
+			return
+		}
+		flash := controller.Flash(session)
+
+		retCode := Code{}
+
+		var form FormData
+		if err := controller.BindForm(w, r, &form); err != nil {
```

Comment:
Instead of using a form, you can use `mux.Vars` and put the UUID in a URL. Just another option here.